graphql_compiler/tests/conftest.py

```diff
@@ -30,25 +33,48 @@ def init_integration_orientdb_client():
     return _init_orientdb_client(load_schema, generate_orient_integration_data)
 
 
+# retry is a decorator that attempts to call the decorated function repeatedly, and it takes a
+# parameter "call". Call is the function being decorated, so we don't need to specify that in
+# the parameters that the retry decorator takes, so we can disable pylint's warning about
+# missing parameters here.
+@retry(20, timeout=1)  # pylint: disable=no-value-for-parameter
 def _init_orientdb_client(load_schema_func, generate_data_func):
+    """Set up a database and return a client that can query the database."""
     graph_name = 'animals'
+    orientdb_client = get_test_orientdb_graph(graph_name, load_schema_func, generate_data_func)
+    return orientdb_client
 
-    # Try to set up the database for the test up to 20 times before giving up.
-    set_up_successfully = False
-    for _ in range(20):
-        try:
-            orientdb_client = get_test_orientdb_graph(graph_name, load_schema_func,
-                                                      generate_data_func)
-            set_up_successfully = True
-            break
-        except Exception as e:  # pylint: disable=broad-except
-            sys.stderr.write(u'Failed to set up test DB: {}'.format(e))
-            time.sleep(1)
-
-    if not set_up_successfully:
-        raise AssertionError(u'Failed to set up database without raising an exception!')
 
-    return orientdb_client
+@pytest.fixture(scope='session')
+def init_integration_neo4j_client():
+    """Return a client for an initialized db, with all test data imported."""
+    return _init_neo4j_client(generate_neo4j_integration_data)
+
+
+# We can disable pylint's warning about missing parameters here because retry is a decorator. See
+# _init_orientdb_client function comment.
+@retry(20, timeout=1)  # pylint: disable=no-value-for-parameter
+def _init_neo4j_client(generate_data_func):
+    """Set up a database and return a client that can query the database."""
+    graph_name = 'animals'
```

Comment:
I'm open to defining `graph_name` somewhere else once instead of in every function (since we want the integration tests' database to be as similar as possible). Do we have a file to put this, or just leave this as a global variable in this file?

Reply:
I can't think of a better place to put this than as a global variable here.